packages/postgresml-pypgrx/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+use pgrx::*;
+use pgrx_pg_sys::info;
+use pyo3::prelude::*;
+
+/// Formats the sum of two numbers as string.
+#[pyfunction]
+fn insert_logs(project_id: i64, model_id: i64, logs: String) -> PyResult<String> {
+    
+    let id_value = Spi::get_one_with_args::<i64>(
+        "INSERT INTO pgml.logs (project_id, model_id, logs) VALUES ($1, $2, $3::JSONB) RETURNING id;",
```

Comment:
Did we include a migration for this table somewhere? We need to make sure it's created on all databases running PostgresML.

Reply:
Yes, need to add the following three to our migration once we freeze on the version number.
```
ALTER TYPE pgml.task ADD VALUE IF NOT EXISTS 'conversation';
ALTER TYPE pgml.task ADD VALUE IF NOT EXISTS 'text_pair_classification';
CREATE TABLE IF NOT EXISTS pgml.logs (
    id SERIAL PRIMARY KEY,
    model_id BIGINT,
    project_id BIGINT,
    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
    logs JSONB
);
```